pages/enterprise-features/workflows.mdoc

```diff
@@ -14,7 +14,7 @@ Each Workflow is made up of a single trigger and one or more actions. Triggers i
 ### Triggers
 
 * Minutes/Hours/Days before an event starts
-* When an event is cancelled
+* When an event is canceled
```

Comment:
we want to use American English, but it is misspelled in a lot of other places as well